src/Microsoft.Health.Fhir.Store.Utils/ExceptionExtention.cs

```diff
@@ -0,0 +1,101 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+using System;
+
+namespace Microsoft.Health.Fhir.Store.Utils
+{
+    public static class ExceptionExtention
```

Comment:
Why is this extending base exception? These all look like SQL specific issues. Would the SqlException be a better target?

Reply:
It seems that not all exceptions are SqlExceptions, for more details, you can ask @SergeyGaluzo